e2e_deployment_files/connectivity_deployment.arm32.template.json

```diff
@@ -0,0 +1,771 @@
+{
+  "modulesContent": {
+    "$edgeAgent": {
+      "properties.desired": {
+        "schemaVersion": "1.0",
+        "runtime": {
+          "type": "docker",
+          "settings": {
+            "minDockerVersion": "v1.25",
+            "loggingOptions": "",
+            "registryCredentials": {
+              "rc1": {
+                "username": "<CR.Username>",
+                "password": "<CR.Password>",
+                "address": "<Container_Registry>"
+              }
+            }
+          }
+        },
+        "systemModules": {
+          "edgeAgent": {
+            "type": "docker",
+            "env": {
+              "UpstreamProtocol": {
+                "value": "<UpstreamProtocol>"
+              },
+              "experimentalfeatures__enabled": {
+                "value": "true"
+              },
+              "experimentalfeatures__enableMetrics": {
+                "value": "true"
+              },
+              "ExperimentalFeatures__EnableUploadLogs": {
+                  "value": "true"
+              }
+            },
+            "settings": {
+              "image": "<Container_Registry>/microsoft/azureiotedge-agent:<EdgeRuntime.BuildNumber>-linux-<Architecture>",
+              "createOptions": "{\"ExposedPorts\": {\"9600/tcp\": {}}, \"HostConfig\": {\"PortBindings\": {\"9600/tcp\": [{\"HostPort\": \"9600\"}]}}}"
+            }
+          },
+          "edgeHub": {
+            "type": "docker",
+            "settings": {
+              "image": "<Container_Registry>/microsoft/azureiotedge-hub:<EdgeRuntime.BuildNumber>-linux-<Architecture>",
+              "createOptions": "{\"ExposedPorts\": {\"9600/tcp\": {}}, \"HostConfig\": {\"PortBindings\": {\"8883/tcp\": [{\"HostPort\": \"8883\"}],\"443/tcp\": [{\"HostPort\": \"443\"}],\"5671/tcp\": [{\"HostPort\": \"5671\"}],\"9600/tcp\": [{\"HostPort\": \"9601\"}]}}}"
+            },
+            "env": {
+              "CollectMetrics": {
```

Comment:
Why do you need another deployment file for arm32?  If we are going to have arm32 deployment file, you can define "optimize_performance_flag" here and no need to change connectivity.sh (90% sure).  But use different deployment json will introduce overhead when changes in test modules need to update in 2 places.  Try to use same deployment file first if possible.

Reply:
Oh you're right. I removed a lot of my debugging feature out. Now they are basically the same. Let me fix it.